COBOL Programming Course #2 - Learning COBOL/COBOL Programming Course #2 - Learning COBOL.md

```diff
@@ -42,21 +42,31 @@ Continuously updated to incorporate modernized and proven programming paradigms
 
 ## How is COBOL being used today?
 
-COBOL is everywhere. You have probably used an application written in COBOL today. For example, consider the following statistics:
+COBOL is omnipresent, and it's highly likely that you've interacted with a COBOL application today. Let's take a look at some compelling statistics:
 
-- About 95% of ATM swipes use COBOL code.
+- Approximately 95% of ATM transactions rely on COBOL codes.
+- COBOL powers 80% of face-to-face transactions.
+- Each day, COBOL systems facilitate a staggering $3 trillion in commerce.
 
-- COBOL powers 80% of in-person transactions.
+To truly grasp the extent of COBOL's prevalence, consider these mind-boggling facts:
 
-- Every day, COBOL systems handle $3 trillion in commerce.
+- On a daily basis, there are 200 times more COBOL transactions executed than there are Google searches.
+- Presently, there are over 250 billion lines of actively running COBOL programs, accounting for roughly 80% of the world's actively utilized code.
+- Every year, approximately 1.5 billion lines of new COBOL code are written.
 
-How pervasive is COBOL? Consider these mind-boggling facts:
+### Highlights from OMP CWG's survey on COBOL in 2021.
 
-- Every day there are 200 times more COBOL transactions executed than there are Google searches.
+Highlights from the survey conducted by OMP CWG (Open Mainframe Project's COBOL Working Group) shed further light on the use of COBOL in today's era:
```

Comment:
Could you remove the OMP CWG from all? I believe the Linux Foundation also have a Open Manufacturing Project and we don't want people to be confused.

So, something like "conducted by the Open Mainframe Project's COBOL Working Group shed further" would be great.